Tests/Scenarios/Server_Scenario_03_Remove.ps1

```diff
@@ -1,3 +1,28 @@
+function Get-CurrentSSLBinding {
```

Comment:
I'm not a huge fan of this being here, but it works... 
(Feels like it should be extracted somewhere else, like a shared module or something).
_Not saying we should change it as part of this PR though_